### PR TITLE
new: add `Troubleshooting` Guides + Dedicated `Metrics` section

### PR DESCRIPTION
**Pull request by incertum**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind user-interface

/kind content

> /kind translation

> /kind event

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area blog

/area documentation

> /area community

**What this PR does / why we need it**:

Add new help guides.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: content/en/docs/install-operate/help/start-up-error.md
Comment (incertum) [inline content/en/docs/install-operate/help/start-up-error.md:7]: @Andreagit97 and @mikegcoleman is this complementing the planned revamp of the Install and Operate Guides. Would you have additional suggestions?

File: content/en/docs/install-operate/help/dropping.md
Comment (incertum) [inline content/en/docs/install-operate/help/dropping.md:2]: @FedeDP 😅 we finally have it ...

File: content/en/docs/install-operate/help/missing-fields.md
Comment (incertum) [inline content/en/docs/install-operate/help/missing-fields.md:21]: @leogr I really hope we make this clearer, I'll post in one of the open issues that perhaps we should reconsider adding `host` to the image if it's a host process and for Falco 0.38 even `pod_sandbox_container` once the relevant libs PR is merged ...

File: content/en/docs/install-operate/help/missing-fields.md
Comment (incertum) [inline content/en/docs/install-operate/help/missing-fields.md:35]: We updated them in libs for improved technical clarity already a while ago.

File: content/en/docs/install-operate/production-performance.md
Comment (incertum) [inline content/en/docs/install-operate/production-performance.md:78]: @Andreagit97 😅 finally ... it's up!

**Comment by leogr** [comment]
cc @LucaGuerra @mikegcoleman

File: content/en/docs/install-operate/help/dropping.md
Comment (Issif) [inline content/en/docs/install-operate/help/dropping.md:166]: why some `*`?

File: content/en/docs/install-operate/help/start-up-error.md
Comment (Issif) [inline content/en/docs/install-operate/help/start-up-error.md:36]: User can explore this site to know which drivers are available https://download.falco.org/driver/site/index.html

Comment (incertum) [reply content/en/docs/install-operate/help/start-up-error.md]: @Issif added, btw nice suggestion!

**Comment by incertum** [comment]
Re-did the metrics fields explanations section, so that it is less crowded and more organized.

**Comment by incertum** [comment]
@leogr and @mikegcoleman changed my mind and create 2 new top level sections, namely "Metrics" and "Help!". Is it better?

**Comment by incertum** [comment]
As we are deciding to start deprecating the old drop stats / alerts we could consider removing https://falco.org/docs/event-sources/kernel/dropped-events/ once the metrics page is up. WDYT?

**Comment by incertum** [comment]
Also https://falco.org/docs/event-sources/kernel/dropped-events/#actions-rate-throttling is actually already removed CC @Andreagit97

**Comment by leogr** [comment]
> As we are deciding to start deprecating the old drop stats / alerts we could consider removing https://falco.org/docs/event-sources/kernel/dropped-events/ once the metrics page is up. WDYT?

Keep this on hold for now.

**Comment by mikegcoleman** [comment]
@incertum should this be /area docs instead of /area blog?

**Comment by mikegcoleman** [comment]
@incertum I think it'd be better if the top level topic was "troubleshooting" instead of "help" - when I see help I think of in-app help, which is basically what documentation is so it seemed a little odd.

**Comment by incertum** [comment]
> @incertum I think it'd be better if the top level topic was "troubleshooting" instead of "help" - when I see help I think of in-app help, which is basically what documentation is so it seemed a little odd.

Happy to change it if everyone is on board. On the other hand you often have help slack channels. Not sure what the best naming would be.

**Comment by leogr** [comment]
> > @incertum I think it'd be better if the top level topic was "troubleshooting" instead of "help" - when I see help I think of in-app help, which is basically what documentation is so it seemed a little odd.
> 
> Happy to change it if everyone is on board. On the other hand you often have help slack channels. Not sure what the best naming would be.

I usually assume the whole document is already a "help" guide. So "troubleshooting" sounds a bit better for me. No strong opinion anyway.

**Review by leogr — APPROVED** [review]

**Pull request merged**
